docs/visualize/vega.asciidoc

```diff
@@ -8,42 +8,46 @@ into Kibana, either standalone, or on top of a map.  To see Vega in action,
 watch this
 https://www.youtube.com/watch?v=lQGCipY3th8[short introduction video].
 
-Currently Vega version 4.4 and VegaLite version 2.6 are supported.
+Currently Vega version 4.4 and Vega-Lite version 2.6 are supported.
 
 NOTE: In Vega it is possible to load data dynamically, e.g. by setting signals as data urls. This is not supported in Kibana as all data is fetched at once prior to passing it to the Vega renderer.
 
 
 [[vega-quick-demo]]
 === Getting Started with Vega
 
-Follow these steps to create your first Vega visualization.
-
-. In Kibana, choose Visualize, and add Vega visualization. You should see a default graph.
-. Try changing `mark` from `line` to `point`, `area`, `bar`, `circle`,
-`square`, ... (see
-https://vega.github.io/vega-lite/docs/mark.html#mark-def[docs])
-. Try other https://vega.github.io/vega/examples/[Vega] or
-https://vega.github.io/vega-lite/examples/[VegaLite] visualizations. You
-may need to make URLs absolute, e.g. replace
+* To experiment using sample data, first click the {kib} logo in the upper left hand corner
+and then click the link next to *Sample Data*.
+* Once you have data loaded, go to *Visualize*, click *+*, and select *Vega* to see an example graph. 
+*Note*: The default graph is written in Vega-Lite, but you can build visualizations in either of the languages (see below for more information). Edit the `schema` value in the dev tools to the left of the graph to define which language you would like to use. For example: `$schema: https://vega.github.io/schema/vega-lite/v2.json`
```

Comment:
I like how you described it, thanks!  How about we move that text to the section below, and link to it?
```suggestion
*Note*: The default graph is written in Vega-Lite, but you can build visualizations in either Vega or Vega-Lite languages (see [#vega-vs-vegalite] section for more information).
```

I updated the below section to

Reply:
P.S. The above might not link correctly to the asciidoc section - not sure what is the proper linking required.

Reply:
Sounds good, updated this link.